bitmapcontainer.go

```diff
@@ -20,8 +22,8 @@ func (bc bitmapContainer) String() string {
 
 func newBitmapContainer() *bitmapContainer {
 	p := new(bitmapContainer)
-	size := (1 << 16) / 64
-	p.bitmap = make([]uint64, size, size)
+	size := (1 << 16) >> log2WordSize
```

Comment:
Do you expect this to be faster?

Reply:
Seems like its faster right?

Reply:
Yes. Both `size := (1 << 16) / 64` and `size := (1 << 16) >> log2WordSize`. are strictly equivalent. We can check the assembly.

Reply:
> Seems like its faster right?

I have not seriously looked at your benchmark numbers, but we need a rationale for why it is faster. Introducing a branch, for example, may make a benchmark appear to be faster, but that can be misleading.

There may be places where introducing a shift as opposed to a division by a compile-time power of two could be faster, mostly due to signed variables...

But the Go compiler most certainly can make these optimizations.

Reply:
I'll try to take a deeper look this week why this might be the case. I'll see if there is a case for the signed variables. If so then I will change the code accordingly.